.github/workflows/pipeline.yml

```diff
@@ -20,184 +20,184 @@
-          cat > vHive.json <<- EOM
-          [
-          {
-              "GatewayID": "10.96.0.4:50051",
-              "FunctionMemoryMB": 128,
-              "ImageSizeMB": 60,
-              "PackageType": "Image"
-          }
-          ]
-          EOM
-          docker run -d --name=chameleon-func --net=benchnet --ip 10.96.0.4 vhiveease/chameleon:latest
-          ./main --g . --o latency-samples --c experiments/tests/vhive/burstiness-image-package.json
-
-      - name: vHive Hello Python Integration Tests
-        run: |
-          cat > vHive.json <<- EOM
-          [
-          {
-              "GatewayID": "10.96.0.5:50051",
-              "FunctionMemoryMB": 128,
-              "ImageSizeMB": 60,
-              "PackageType": "Image"
-          }
-          ]
-          EOM
-          docker run -d --name=hellopy-func --net=benchnet --ip 10.96.0.5 vhiveease/hellopy:latest
-          ./main --g . --o latency-samples --c experiments/tests/vhive/burstiness-image-package.json
-
-      - name: vHive RNN Serving Integration Tests
-        run: |
-          cat > vHive.json <<- EOM
-          [
-          {
-              "GatewayID": "10.96.0.6:50051",
-              "FunctionMemoryMB": 128,
-              "ImageSizeMB": 60,
-              "PackageType": "Image"
-          }
-          ]
-          EOM
-          docker run -d --name=rnnserving-func --net=benchnet --ip 10.96.0.6 vhiveease/rnnserving:latest
-          ./main --g . --o latency-samples --c experiments/tests/vhive/burstiness-image-package.json
-
-  deploy:
-    name: Deploy framework
-    needs: [ test_aws, test_vhive ]
-    runs-on: ubuntu-22.04
-    steps:
-
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Download client build
-        uses: actions/download-artifact@v2
-        with:
-          name: STeLLAR-build
-
-      - name: Untar client build
-        run: tar --strip-components=1 -xvf build.tar
-
-      - name: Prepare benchmarking functions
-        run: |
-          mkdir -p setup/deployment/raw-code/functions
-          cp -R ./src/setup/deployment/raw-code/functions setup/deployment/raw-code/functions
-
-      - name: Push to deployment branch
-        run: |
-          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git config user.name "vHive Bench Bot"
-          git remote set-url origin https://ease-lab:${{secrets.ACCESS_TOKEN}}@github.com/ease-lab/STeLLAR.git
-          git checkout --orphan deployment
-          git rm -rf --ignore-unmatch --cached .
-
-          git add --ignore-errors --force scripts endpoints experiments main setup/deployment/raw-code/functions
-
-          git commit -m "Automated push from successful STeLLAR pipeline"
-          git push origin HEAD:deployment --force
+# name: STeLLAR Pipeline
+# 
+# concurrency: staging_environment
+# 
+# on:
+#   push:
+#     branches: [ main ]
+#   pull_request:
+#     branches: [ main ]
```

Comment:
For `pipeline.yml` do you think it would be better to remove the file entirely instead of commenting it out? Since the original file is preserved in the `main-legacy` branch

Reply:
Good idea, I left it there in case there is future integration of vHive, since `main-legacy` has this file for reference I'll remove it

Reply:
Sure, thanks!